### ISSUE TITLE
[CLI 0.17.19] quasar.conf > build > gzip: true breaks build

### ISSUE THREAD

**Issue by gcallaghan** [quasarframework/quasar-cli#184]
### Software version

OS: OS X
Node:v9.11.1
NPM:6.4.1
Any other software related to your bug:
webpack

### What did you get as the error?
(node:49202) UnhandledPromiseRejectionWarning:   ValidationError: Compression Plugin Invalid Options
### What were you expecting?
build to succeed
### What steps did you take, to get the error?
build as normal

$ quasar build -m pwa

 Build mode........ pwa
 Quasar theme...... mat
 Quasar CLI........ v0.17.19
 Quasar Framework.. v0.17.16
 Debugging......... no

 app:quasar-conf Reading quasar.conf.js +0ms
 app:workbox [GenerateSW] Will generate a service-worker file. Ignoring your custom written one. +7s
 app:webpack Chaining PWA Webpack config +1ms
(node:49202) UnhandledPromiseRejectionWarning:   ValidationError: Compression Plugin Invalid Options
  options should NOT have additional properties

  - validateOptions.js:32 validateOptions
    [ciitizen-ui]/[schema-utils]/src/validateOptions.js:32:11

  - index.js:55 new CompressionPlugin
    [ciitizen-ui]/[compression-webpack-plugin]/dist/index.js:55:31

  - Plugin.js:13 init
    [ciitizen-ui]/[webpack-chain]/src/Plugin.js:13:18

  - Plugin.js:59 Object.toConfig
    [ciitizen-ui]/[webpack-chain]/src/Plugin.js:59:22

  - Config.js:129 clean.Object.assign.plugins.plugins.values.map.plugin
    [ciitizen-ui]/[webpack-chain]/src/Config.js:129:61

  - Array.map

  - Config.js:129 module.exports.toConfig
    [ciitizen-ui]/[webpack-chain]/src/Config.js:129:40

  - index.js:16 getWebpackConfig
    [ciitizen-ui]/[quasar-cli]/lib/webpack/index.js:16:31

  - index.js:45 getPWA
    [ciitizen-ui]/[quasar-cli]/lib/webpack/index.js:45:10

  - index.js:120 module.exports
    [ciitizen-ui]/[quasar-cli]/lib/webpack/index.js:120:12

  - quasar-config.js:739 QuasarConfig.compile
    [ciitizen-ui]/[quasar-cli]/lib/quasar-config.js:739:46
 19   ¦ ¦deploy:prod-guest¦: ¦aws s3  sync --acl public-read --delete ./dist/guest/ s3://prod.ciitizen.ui-guest¦

  - quasar-build:120 build
    [ciitizen-ui]/[quasar-cli]/bin/quasar-build:120:16

  - next_tick.js:182 process._tickCallback
    internal/process/next_tick.js:182:7

  - loader.js:697 Function.Module.runMain
    internal/modules/cjs/loader.js:697:11

  - node.js:201 startup
    internal/bootstrap/node.js:201:19

(node:49202) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:49202) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by rstoenescu** [comment]
How does your quasar.conf > build > gzip property looks like?
The webpack-compression-plugin was upgraded to v2.

**Comment by gcallaghan** [comment]
```
[...snip...]
build: {
      scopeHoisting: true,
      vueRouterMode: 'history',
      gzip: true,
      analyze: false,
      useNotifier: false,
[...snip...]
```

**Comment by rstoenescu** [comment]
Thanks for reporting. Fixed the issue and will be available in CLI v0.17.20.

**Comment by gcallaghan** [comment]
Thanks for the quick fix and working on this in general. It's a great framework/tool!
